Lib/test/ann_module.py

```diff
@@ -51,3 +52,8 @@ def foo(x: int = 10):
     def bar(y: List[str]):
         x: str = 'yes'
     bar()
+
+def dec(func):
+    def wrapper(*args, **kwargs):
+        return func(*args, **kwargs)
+    return wraps(func)(wrapper)
```

Comment:
I think the tests are failing because of this missing newline:

```suggestion
    return wraps(func)(wrapper)

```